.github/workflows/check-traceability.yml

```diff
@@ -0,0 +1,46 @@
+name: Check PR traceability
+
+on:
+  push:
+    branches: [master, "release/*"]
+  pull_request:
+    branches: [master, "release/*"]
+
+jobs:
+  check-pr-description:
+    name: Check PR description
+    runs-on: ubuntu-20.04
+    steps:
+
+    - uses: actions/checkout@v2
+
+    - name: Set up Python 3.6
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.6
+
+    - name: Fetch PR Description
+      env:
+        GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+      run: echo "::set-output name=description::$(gh pr view ${{ github.event.pull_request.number }} | tr -d "\n\r" | tr -d ' ')"
+      id: pr
+
+    - name: Check PR validity
+      env:
+        PR_DESCRIPTION: ${{ steps.pr.outputs.description }}
+      shell: python
+      run: |
+        import os
+        import re
+
+        description = os.getenv("PR_DESCRIPTION").lower()
+
+        if '<issue_number>' in description:
+          raise Exception("The template should be edited and issues tracking should be added.")
+
+        contains_fix = 'fixes#' in description
```

Comment:
Should we also check for `https://github.com/PyTorchLightning/pytorch-lightning/issues/xxxx` in the description?

Reply:
I think it is better to track for fixes, as it is used by Github to automate issue closing based on those words.

Reply:
Missing space between fixes and #
How about other key words like fix, resolve, close, resolves, closes...